easybuild/easyconfigs/m/Mesa/Mesa-21.2.0-GCCcore-11.2.0.eb

```diff
@@ -0,0 +1,69 @@
+# This is a Mesa using software rendering via Gallium-DRI and libglvnd
+# - libglvnd can dynamically choose between system-installed NVidia
+# libGLX/libEGL or the software renderers provided by this Mesa
+# - EGL is available
+#
+# Software renderers are enabled based on system architecture:
+# - llvmpipe: the high-performance Gallium LLVM driver
+# - swr: Intel's OpenSWR
+# Default renderer is llvmpipe. SWR can be enabled by setting the environment
+# variable GALLIUM_DRIVER=swr
+
+name = 'Mesa'
+version = '21.2.0'
```

Comment:
Note: Mesa 21.2.0 is not a stable release, so this was moved to 21.1.7 in #13736